Makefile

```diff
@@ -64,7 +68,7 @@ images: ## Generate most used image variations for all images for specified envi
 
 .PHONY: migrations
 migrations: ## Run Doctrine migrations for specified environment (default: SYMFONY_ENV=dev)
-	$(PHP_RUN) bin/console doctrine:migration:migrate --env=$(SYMFONY_ENV)
+	$(PHP_RUN) bin/console doctrine:migration:migrate --allow-no-migration --env=$(SYMFONY_ENV)
```

Comment:
Not related directly to this PR, but migrations fail if there are no migration files, so I've adjusted this.